### PR TITLE
fix: use zrandmember, get emtpy value when  withscores

### PR DESCRIPTION
**Pull request by lenonqing**

### CODE REVIEW

**Comment by rueian** [comment]
Hi @lenonqing,

Thank you for reporting this. However, it seems that this should be fixed by aligning with go-redis' Cmdable interface:
https://github.com/go-redis/redis/blob/ce016ed85fb76b085fe6c5950a80883a5cd24f89/commands.go#L295-L296

**Comment by lenonqing** [comment]
ok, i want to use` cmds.Completed`,`cmds.Arbitrary`,`cmds.Cacheable`, but i can't access them, can export?^-^
Thanks!

File: cmds.go [hunk 1]
```diff
@@ -17,3 +17,9 @@ import "github.com/rueian/rueidis/internal/cmds"
 //   }
 // However, please know that once commands are processed by the Do() or DoMulti(), they are recycled and should not be reused.
 type Commands []cmds.Completed
+
+type (
+	Arbitrary = cmds.Arbitrary
+	Completed = cmds.Completed
+	Cacheable = cmds.Cacheable
+)
```
Comment (rueian) [inline cmds.go:25]: Could you show me how do you want to access them? These lines look redundant to me.

**Comment by rueian** [comment]
Hi @lenonqing,

The fix is already cherry-picked. I am going to close this PR. Please feel free to open a new one.

**Pull request closed without merging**